pkg/backup/item_backupper_test.go

```diff
@@ -123,6 +123,35 @@ func TestBackupItemSkipsClusterScopedResourceWhenIncludeClusterResourcesFalse(t
 	assert.NoError(t, err)
 }
 
+func TestBackupItemWhenInTerminatingStatus(t *testing.T) {
+	f := true
+	ib := &defaultItemBackupper{
+		backup: &v1.Backup{
```

Comment:
You should be able to do `&v1.Backup{}` - any reason you needed `IncludeClusterResources`?